parquet-format-structures/src/main/java/org/apache/parquet/format/Util.java

```diff
@@ -216,8 +277,41 @@ private static TProtocol protocol(InputStream from) {
   private static InterningProtocol protocol(TIOStreamTransport t) {
     return new InterningProtocol(new TCompactProtocol(t));
   }
+  
+  private static InputStream decryptStream(InputStream from, BlockCipher.Decryptor decryptor) throws IOException {    
```

Comment:
Could you please use more descriptive variable names instead of i32rd, clen, cbuf, pbuf, tmb, plen and i32out?

Reply:
Sure, will do.